marconi-sidechain/README.md

```diff
@@ -1,37 +1,67 @@
 # marconi-sidechain
 
-`marconi-sidechain` is a lightweight chain follower application for the Sidechain project to index and query specific information from the Cardano blockchain.
+`marconi-sidechain` is a lightweight chain-follower application for the Sidechain project to index and query specific information from the Cardano blockchain.
 The interface for querying the indexed information uses [JSON-RPC](https://www.jsonrpc.org/specification) over HTTP.
 
-See the [architecture documentation](./doc/ARCHITECTURE.adoc) for more information on how this application was build.
+See the [architecture documentation](./doc/ARCHITECTURE.adoc) for more information on how this application was built.
 
 ## Prerequisites
 
-If using `Nix`:
+If using **Nix**:
 
 * [Nix](https://nixos.org/download.html) (`>=2.5.1`)
-  * Enable IOHK's binary cache or else you will build the world! Refer to [this section](../CONTRIBUTING.adoc#how-to-get-a-shell-environment-with-tools) on how to achieve this.
+* Ensure you have the following configuration (eg in `~/.config/nix/nix.conf`):
 
-If *not* using `Nix`:
+```
+experimental-features = nix-command flakes
```

Comment:
do we want to provide the CLI option alternative to enable flakes?

Reply:
I see no reason not to enable `nix-command flakes` on every nix system, so I think it's simpler just to tell people to do that. If someone really cares about not enabling them they're almost certainly a nix expert and will know about the CLI option alternative already.

Reply:
True, I was just thinking about the test via a docker image scenario, where it may sound less cumbersome to add a parameter rather than modify a config file. I agree though that the difference is minor and omitting it is probably less confusing.